.gitignore

```diff
@@ -1,7 +1,15 @@
-# Godot-related
+
+# Godot-specific ignores
 .import/
-export_presets.cfg
 
-# System-specific
+# Imported translations (automatically generated from CSV files)
+*.translation
```

Comment:
I'm not sure about that. If `.translation` files are meant to be ignored, they should be put in `.import/`. But let's give it a try.

Reply:
See https://github.com/github/gitignore/blob/master/Godot.gitignore